microprofile.ls/org.eclipse.lsp4mp.ls/src/test/java/org/eclipse/lsp4mp/services/ApplicationPropertiesDiagnosticsTest.java

```diff
@@ -230,6 +230,15 @@ public void validateUnknownPropertiesExcludedWithPattern() throws BadLocationExc
 		// * pattern --> all errors are ignored
 		unknown.setExcluded(new String[] { "*" });
 		testDiagnosticsFor(value, 0, getDefaultMicroProfileProjectInfo(), settings);
+
+		value = "kafka-streams.cache.max.bytes.buffering=10240\n" + //
```

Comment:
As your test doesn't use the original value, I suggest that you create a new test method like `validateUnknownPropertiesWithProfileExcludedWithPattern`

Reply:
Done.